compiler-plugin/src/main/kotlin/com/google/devtools/ksp/processor/CheckOverrideProcessor.kt

```diff
@@ -69,5 +71,11 @@ class CheckOverrideProcessor : AbstractTestProcessor() {
         checkOverride(getY, myInterfaceX)
         checkOverride(getX, myInterfaceX)
         checkOverride(setY, myInterfaceY)
+        checkOverride(getY, getY)
```

Comment:
Can you add a test for return type checking here? Simply change the return type of `fun equals(other: Any?)` to another type such as `Int` shall work.

Reply:
added one and also another one to check overriding val property with setter.
Appearantly that already works because we cannot find a matching property but still wanted to add this.